spec/factories/applications.rb

```diff
@@ -81,5 +81,13 @@
     trait :with_random_user do
       user { FactoryBot.build(:user, :with_random_name) }
     end
+
+    trait :with_participant_id_change do
+      after(:create) do |application|
+        user = application.user
+
+        FactoryBot.create(:participant_id_change, from_participant: user, user:)
```

Comment:
could we amend this to move from a previous other user to the current user? maybe something like this:
`FactoryBot.create(:participant_id_change, to_participant: user, user:)`